opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -338,7 +338,15 @@ namespace Opm
                          const Opm::EclipseState& eclipseState,
                          std::unique_ptr<EclipseIO>&& eclIO,
                          const Opm::PhaseUsage &phaseUsage)
-      : output_( param.getDefault("output", true) ),
+        : output_( [&param](){
+                // If output parameter is true or all, then we do output
+                std::string outputString = param.getDefault("output", std::string("all"));
+                if ( outputString == "all" ||  outputString == "true" )
```

Comment:
Also, if we're sticking to (very) strict C++11, then the lambda needs to be declared as
```C++
[&param]() -> bool { /* stuff */ }
```
because multi-statement lambdas don't have deduced return types in strict C++11 (return type deduction is reserved for single statement bodies of the exact form `return expression;` in this strict dialect).

On the other hand, most (relevant) compilers implement the defect report that supports return-type deduction for multi-statement bodies even in C++11 mode, so this may not matter too much in practice.